A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_WHEEL.js

```diff
@@ -0,0 +1,397 @@
+            this.spoilerLeft.setAttribute("visibility","hidden");
+
+            this.tempText = this.querySelector("#temp");
+            this.tempText.setAttribute("visibility","hidden");
+
+            // Need to finish making failure logic
+            this.failureLGCIUS1 = false;
+            this.failureLGCIUS2 = false;
+
+            // End of Failures
+
+            this.dummyTempCounter = 1;
+            this.dummyTempCounterCoolDown = 1;
+
+            // The code below can be stream line better for now it works 
+            this.currentBreak1Temp = 45;
+            this.currentBreak2Temp = 50;
+            this.currentBreak3Temp = 45;
+            this.currentBreak4Temp = 50;
+
+            // Set Default State of Brakes Temp
+            this.BreakTemp1.textContent = 45;
+            this.BreakTemp2.textContent = 50;
+            this.BreakTemp3.textContent = 45;
+            this.BreakTemp4.textContent = 50;
+
+                
+            // - Some variables holders 
+            this.engineOneHydG = "G";
+            this.engineOneHydY = "Y";
+            this.autoBrakeLow = "LOW";
+            this.autoBrakeMed = "MED";
+            this.autoBrakeMax = "MAX";
+
+
+            //LandingGEAR
+            this.leftLGGroup = this.querySelector("#leftSideLandingGear");
+            this.rightLGGroup = this.querySelector("#RightSideLandingGear");
+            this.centerLGGroup = this.querySelector("#centerLandingGroup");
+
+            this.LGFailureXX = this.querySelector("#LGFailure");
+            this.LGFailureXX.setAttribute("visibility","hidden");
+
+            this.greenBarLeft = this.querySelector("#leftLGBar");
+            this.greenBarRight = this.querySelector("#rightLGBar");
+            this.greenBarCenter = this.querySelector("#LGFailure");
+
+            this.orangeBars = this.querySelector("#orangeLGDown");
+            this.orangeBars.setAttribute("visibility","hidden");
+
+            this.greenBarUp = this.querySelector("#greenBarUp");
+
+        }   
+
+
+        update(_deltaTime) {
+            if (!this.isInitialised) {
+                return;
+            }
+            this.updateTempColor(_deltaTime);
+            this.updateHydroSymbols(_deltaTime);
+            this.updateAutoBreak(_deltaTime);
+            this.updateSkidToggle(_deltaTime);
+            this.updateSpoilerSpeedBreak(_deltaTime);
+            this.updateLandingGear(_deltaTime);
+
+
+            // Break Temp - this is a based loop checking both speed and current break pressure
+            // 
+            const currentBreakRight = SimVar.GetSimVarValue("BRAKE RIGHT POSITION", "position 32k");
+            const currentBreakLeft =  SimVar.GetSimVarValue("BRAKE LEFT POSITION", "position 32k");
+            const currentParkingBreak = SimVar.GetSimVarValue("BRAKE PARKING INDICATOR","Bool");
+
+            if(Simplane.getGroundSpeed() > 20 && currentBreakLeft >= 30000 && currentBreakRight >= 30000 && !currentParkingBreak)
```

Comment:
```suggestion
            if(Simplane.getGroundSpeed() > 20 && currentBreakLeft >= 30000 && currentBreakRight >= 30000 && !currentParkingBreak)
```

Might be worth looking into using the WHEEL RPM simvar rather than ground speed to calculate brake temps.

Reply:
Ground was use cause there's no way of really "cooling" off the breaks so I figure it would be best to hard code a limit to at which point break temp is being applied based on speed , once the brake fan switch is added i can switch over code / logic to use RPM